ddtrace/bootstrap/sitecustomize.py

```diff
@@ -43,6 +43,22 @@ def update_patched_modules():
         EXTRA_PATCHED_MODULES.update({module: should_patch.lower() == 'true'})
 
 
+def add_global_tags(tracer):
```

Comment:
Like @brettlangdon mentioned, we don't need to do any convert logic since the tag will be converted back to a string.